test/unit/lib.js

```diff
@@ -403,3 +403,22 @@ test('should loop through each element of an array', function() {
     }
   });
 });
+
+//getFileExtension tests
+test('should get the file extension of the passed path', function() {
```

Comment:
A protocol-relative URL (e.g. "//example.com/movie.mp4") would be nice but I don't think it needs to hold up merging this.